Makefile

```diff
@@ -71,6 +71,8 @@ prow-secrets:
 	oc create secret generic cluster-secrets-gcp --from-file=cluster/test-deploy/gcp/gce.json --from-file=cluster/test-deploy/gcp/ssh-privatekey --from-file=cluster/test-deploy/gcp/ssh-publickey --from-file=cluster/test-deploy/gcp/ops-mirror.pem -o yaml --dry-run | oc apply -f -
 	# gce.json is used by jobs operating against AWS
 	oc create secret generic cluster-secrets-aws --from-file=cluster/test-deploy/aws/.awscred --from-file=cluster/test-deploy/aws/pull-secret --from-file=cluster/test-deploy/aws/license --from-file=cluster/test-deploy/aws/ssh-privatekey --from-file=cluster/test-deploy/aws/ssh-publickey -o yaml --dry-run | oc apply -f -
+	# azure secrets usef by jobs operating agains Azure
+	oc create secret generic cluster-secrets-azure --from-file=cluster/test-deploy/azure/secret -o yaml --dry-run | oc apply -f -
```

Comment:
Example file committed it. We need create dedicated SP on azure for this.

Reply:
`cluster/test-deploy/azure/secret_example` uses exports whereas here we want to create a k8s secret. I would get rid of the example file and create the secret here using env variables. Also move into the azure target.

Reply:
That works too (same as we have now for purge).